Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityFocusProvider.cs

```diff
@@ -7,6 +7,11 @@
 
 namespace Microsoft.MixedReality.Toolkit.Input
 {
+    /// <summary>
+    /// Delegate type used to handle primary pointer changes
+    /// </summary>
```

Comment:
Thoughts about adding some remarks about possible transitions include null->non-null, pointerA to pointer B, non-null->null? Or is this just generally understood by folks using this?